tests/unit_tests/test_bi_superset_adaptor.py

```
@@ -0,0 +1,1098 @@
+    _normalize_series_columns_in_query,
+    _normalize_series_columns_in_query_context,
+    _parse_datasource_value,
+)
+
+
+@pytest.fixture
+def auth_params():
+    """Create test authentication parameters."""
+    return AuthParam(
+        username="test_user",
+        password="test_password",
```

Ruff (0.14.11)

Possible hardcoded password assigned to argument: "password"

(S106)